class.jetpack.php

```diff
@@ -2703,7 +2703,8 @@ function admin_banner_styles() {
 		$min = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
 
 		wp_enqueue_style( 'jetpack', plugins_url( "css/jetpack-banners{$min}.css", JETPACK__PLUGIN_FILE ), false, JETPACK__VERSION . '-20121016' );
-		$wp_styles->add_data( 'jetpack', 'rtl', true );
```

Comment:
Since we're removing `$wp_styles` from the scope of this function, let's also remove the `global $wp_styles` declaration.